### PR TITLE
fix Boolean 'false' value is blank in csv

### PR DESCRIPTION
**Pull request by codeewander**
Fix issue #145

### CODE REVIEW

File: src/core.js [hunk 1]
```diff
@@ -47,7 +47,13 @@ export const getHeaderValue = (property, obj) => {
   return (foundValue === undefined) ? '' : foundValue;
 }
 
-export const elementOrEmpty = (element) => element || element === 0 ? element : '';
+export const elementOrEmpty = element => {
+  if (element == null) {
+    if (element === 0) return element
```
Comment (mccabemj) [inline src/core.js]: If the element is null, it can't then be 0

**Comment by andrewalkema** [comment]
Any progress on this one?  This is the one thing keeping me from using this.

**Comment by AndreVizosodaCruz** [comment]
Hello any update on this issue?

**Comment by mccabemj** [comment]
#193 has been created to replace this

**Pull request closed without merging**